mapadroid/utils/resolution.py

```diff
@@ -66,24 +66,20 @@ def get_delete_quest_coords(self):
             click_x = int(self._screen_x) / 1.07
         return click_x + self._x_offset, click_y + self._y_offset
 
-    def get_swipe_item_amount(self):
+    def get_click_item_minus(self):
```

Comment:
`click_x = int(self._screen_x) / 3.7` could come before the if/else blocks to avoid code-duplication